src/components/HeaderWithBackButton/headerWithBackButtonPropTypes.js

```diff
@@ -89,6 +89,9 @@ const propTypes = {
 
     /** Policies, if we're showing the details for a report and need participant details for AvatarWithDisplay */
     personalDetails: PropTypes.objectOf(participantPropTypes),
+
+    /** Children to wrap in Header */
+    children: PropTypes.node,
```

Comment:
I wonder if defining children in the propTypes here will conflict with [this declaration](https://github.com/Expensify/App/blob/21d14c5135d186f2f3ba9dc0e8d53f071d43f80d/src/components/IllustratedHeaderPageLayout.js#L14-L17)

Reply:
Also, children is not required but we're not setting a default value for it

Reply:
Making it required has been complaining in the logs as some usecases were not passing children I think